src/gists/gist.jl

```diff
@@ -0,0 +1,79 @@
+type Gist <: GitHubType
+    url::Nullable{HttpCommon.URI}
+    forks_url::Nullable{HttpCommon.URI}
+    commits_url::Nullable{HttpCommon.URI}
+    id::Nullable{GitHubString}
+    description::Nullable{GitHubString}
+    public::Nullable{Bool}
+    owner::Nullable{Owner}
+    user::Nullable{Owner}
+    truncated::Nullable{Bool}
+    comments::Nullable{Int}
+    comments_url::Nullable{HttpCommon.URI}
+    html_url::Nullable{HttpCommon.URI}
+    git_pull_url::Nullable{HttpCommon.URI}
+    git_push_url::Nullable{HttpCommon.URI}
+    created_at::Nullable{Dates.DateTime}
+    updated_at::Nullable{Dates.DateTime}
+    forks::Nullable{Vector{Gist}}
+    files::Nullable{Dict}
+    history::Nullable{Vector{Dict}}
```

Comment:
You can just add the field with the type you want (e.g. the [`Owner` field in the `Repo` type definition](https://github.com/JuliaWeb/GitHub.jl/blob/dc6d368b1a9ad826676919f816fd4ba27b2e982a/src/repositories/repositories.jl#L5)). The JSON-to-Julia-object conversion [is hackily handled automagically](https://github.com/JuliaWeb/GitHub.jl/blob/dc6d368b1a9ad826676919f816fd4ba27b2e982a/src/utils/GitHubType.jl#L80).